Mathlib/Algebra/Category/GroupCat/EnoughInjectives.lean

```diff
@@ -0,0 +1,49 @@
+/-
+Copyright (c) 2023 Jujian Zhang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jujian Zhang, Junyan Xu
+-/
+
+import Mathlib.Algebra.Module.CharacterModule
+import Mathlib.Algebra.Category.GroupCat.EquivalenceGroupAddGroup
+
```

Comment:
The linter complains that `import Mathlib.Algebra.Category.GroupCat.EpiMono` in CharacterModule.lean needs to be moved here.